bump-release.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+from cid._version import __version__ as old_ver
+
+
+bump='patch'
+if len(sys.argv)>1 :
+	bump = sys.argv[1]
+
+maj, minor, patch = map(int, old_ver.split('.'))
+
+if bump=='patch':
+	new_ver = '.'.join(map(str,[maj,minor,patch + 1]))
+elif bump=='minor':
+	new_ver = '.'.join(map(str,[maj,minor + 1,patch]))
```

Comment:
If we bump minor or major version number we have to set lower numbers to 0

Reply:
yes good catch